mzLib/FlashLFQ/FlashLFQResults.cs

```diff
@@ -17,7 +17,7 @@ public class FlashLfqResults
         public readonly Dictionary<string, ProteinGroup> ProteinGroups;
         public readonly Dictionary<SpectraFileInfo, List<ChromatographicPeak>> Peaks;
         private readonly HashSet<string> _peptideModifiedSequencesToQuantify;
-        public IDictionary<string, Dictionary<PeakRegion, List<ChromatographicPeak>>> IsobaricPeptideDict = null;
+        public IDictionary<PeptideMassBin, Dictionary<PeakRegion, List<ChromatographicPeak>>> IsobaricPeptideDict = null;
```

Comment:
I think this should be PeptideMzBin unless it is deconvoluted neutral mass. In that case, Mass is fine.